hedera-platform/build.gradle.kts

```diff
@@ -72,10 +76,12 @@ dependencies.constraints {
         api(gav("io.helidon.grpc.client", helidonVersion))
         api(gav("io.helidon.grpc.core", helidonVersion))
         api(gav("io.helidon.grpc.server", helidonVersion))
+        api(gav("io.netty.codec.http2", nettyVersion))
```

Comment:
We probably shouldn't pull this in separately.  Helidon and GRPC both specify the required version, and we don't use this library directly.

Reply:
This was added prior to the enforced netty BOM, so this can likely be removed.

Reply:
Removed without issue since we have the netty BOM enforced.